torch/csrc/dynamo/eval_frame.c

```diff
@@ -727,6 +727,24 @@ static int clear_state(PyObject* module) {
 
 bool is_skip_guard_eval_unsafe = false;
 
+int c_recursion_limit = -1;
```

Comment:
You don't want anyone accessing it directly, do you?
```suggestion
static int c_recursion_limit = -1;
```